app/controllers/api/v1/countries_controller.rb

```diff
@@ -27,7 +29,39 @@ def set_country
   def set_countries
     @countries = GeoEntity.countries
       .order(name: :asc)
-      .paginate(page: params[:page] || 1, per_page: 25)
+      .paginate(page: page, per_page: per_page)
       .eager_load(geo_entity_stats: :habitat)
   end
+
+  def countries_as_json
+    @countries.as_json(
+      only: %i[name iso3],
+      methods: %i[protected_area_statistics]
+    )
+  end
+
+  def index_metadata
+    {
+      page: page,
+      per_page: per_page,
+      page_count: page_count,
+      total_count: total_count
+    }
+  end
+
+  def page
+    params[:page] || 1
+  end
+
+  def per_page
+    25
```

Comment:
unless we're allowing a param here, should probably be a constant rather than a method